components/raftstore/src/store/snap.rs

```diff
@@ -1791,24 +1802,22 @@ pub mod tests {
 
     #[test]
     fn test_empty_snap_file() {
-        test_snap_file(open_test_empty_db, None);
-        test_snap_file(open_test_empty_db, Some(gen_db_options_with_encryption()));
```

Comment:
Why remove this?

Reply:
I adjust the testing code to match how we actually use encryption in tikv. After that, this test logic doesn't work anymore. I believe we shouldn't call `Snapshot::new_xx` directly to make sure encryption works consistently.